sdk/communication/Azure.Communication.CallingServer/tests/CallingServerClient/CallingServerClientLiveTests.cs

```diff
@@ -91,13 +93,14 @@ public async Task RunCreatePlayCancelHangupScenarioTests()
             try
             {
                 var callLocator = new GroupCallLocator(groupId);
+                await WaitForOperationCompletion().ConfigureAwait(false);
```

Comment:
no need to wait after creating calllocators. pl fix at other places in code as well